src/utils/error/errors.js

```diff
@@ -38,6 +38,11 @@ errorInfo[(exports.MESSAGE_NOT_EXIST = "MessageIDNotExistError")] = {
   message: "Error: message with corresponding ID does not exist"
 };
 
+errorInfo[(exports.MESSAGE_NO_TEXT = "MessageNotContainText")] = {
```

Comment:
Should this be called MESSAGE_DOES_NOT_EXIST? I think we should use 404 for not found.

Reply:
no this is really for a malformed request

Reply:
id probably should have put this somewhere in the validator